crates/cast/src/starknet_commands/account/add.rs

```diff
@@ -99,3 +122,119 @@ pub async fn add(
         },
     })
 }
+
+fn import_from_keystore(
+    keystore_path: &Utf8PathBuf,
+    account_path: &Utf8PathBuf,
+) -> Result<serde_json::Value> {
+    let account_info: serde_json::Value = serde_json::from_str(&fs::read_to_string(account_path)?)?;
+    let deployment = account_info
+        .get("deployment")
+        .ok_or_else(|| anyhow!("No deployment field in account JSON file"))?;
```

Comment:
theoritically - is it possible? or is it that if it's not present, it means the json file is corrupt/invalid?